src/index.js

```diff
@@ -1,13 +1,11 @@
 import 'normalize.css';
-import './sass/style.scss';
+// import './sass/style.scss';
```

Comment:
Don't forget to re-enable this. Your code wouldn't compile without disabling this.